src/microbe_stage/Microbe.cs

```diff
@@ -907,7 +907,7 @@ public List<(Compound Compound, Color Colour, Vector3 Target)> GetDetectedCompou
         var detections = new List<(Compound Compound, Color Colour, Vector3 Target)>();
         foreach (var (compound, range, minAmount, colour) in activeCompoundDetections)
         {
-            var detectedCompound = clouds.FindCompoundNearPoint(Translation, compound, range, minAmount);
+            var detectedCompound = clouds.FindCompoundNearPoint(GlobalTranslation, compound, range, minAmount);
```

Comment:
GlobalTranslation might be a bit expensive to calculate as it needs to find all parent nodes up to the scene root, so having a variable to hold this outside the loop would be nice.

Reply:
I moved it outside the loop.